src/seedu/addressbook/AddressBook.java

```diff
@@ -145,6 +145,14 @@
     private static final int PERSON_DATA_INDEX_PHONE = 1;
     private static final int PERSON_DATA_INDEX_EMAIL = 2;
 
+    /*
+     * Program arguments
+     * */
+    private static final int INVALID_PROGRAM_ARGS = 2;
+    private static final int GIVEN_FILE_STORAGE_PROGRAM_ARGS = 1;
+    private static final int DEFAULT_FILE_STORAGE_PROGRAM_ARGS = 0;
```

Comment:
Awesome, you remove the magic number and use good name to replace those constants .